pysb/tests/test_kappa.py

```diff
@@ -197,13 +198,16 @@ def test_influence_map_kasa():
 def test_unicode_strs():
     Monomer(u'A', [u'b'], {u'b':[u'y', u'n']})
     Monomer(u'B')
-    Rule(u'rule1', A(b=u'y') + KappaDot() >> KappaDot() + B(),
+    Rule(u'rule1', A(b=u'y') + None >> None + B(),
```

Comment:
This is overloading an unrelated test. The `None` testing ought to live in its own test case.

Reply:
Done.